Marlin/Marlin_main.cpp

```diff
@@ -782,7 +782,7 @@ extern "C" {
 #endif // !SDSUPPORT
 
 #if ENABLED(DIGIPOT_I2C)
-  extern void digipot_i2c_set_current(int channel, float current);
+  extern void digipot_i2c_set_current(uint8_t channel, float current);
```

Comment:
> Is Global variables the SRAM ? If not, how do you guys check that ?

Yes.  The AVR parts we are using have 8KB of Static RAM inside them.   The Global variables are always present.  The space for them is 'hard allocated'.  They can be accessed from any code that knows how to reach out and touch them.    

The local variable space is what is left over out of the 8KB or RAM.  The stack lives in that space.  And any local variables defined at the start of function live on the stack.  They go away when the function exits.